docs/guide/tutorial-4-more-about-controllers-20-minutes.md

```diff
@@ -6,19 +6,16 @@ We will design a simple controller that avoids the obstacles created in the prev
 This tutorial will introduce you to the basics of robot programming in Webots.
```

Comment:
We should clearly write this tutorial in all the languages (like it is done in for the API), but this should be done in another PR.

Reply:
We should probably also use another robot as the e-puck is one of our worst nice-looking robot.

Reply:
This will be done in a subsequent PR.